ohmec_data_na.js

```diff
@@ -4,6 +4,147 @@ dataNA = {
+      "geometry":{
+        "type":"Polygon",
+        "coordinates":[[[-122.5, 38],[-122.5, 36.5],[-120.5, 36.5],[-120.5, 38]]]}},
+    { "type":"Feature",
+      "id":"ES700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Indigenous",
+        "entity2type":"tribe",
+        "entity2name":"Esselen",
+        "labelRotate":45,
+        "labelScale":2.5,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Esselen",
+        "startdatestr":"700",
+        "enddatestr":"1768"},
+      "geometry":{
+        "type":"Polygon",
+        "coordinates":[[[-122, 36.5],[-122, 36],[-121.5, 36],[-121.5, 36.5]]]}},
+    { "type":"Feature",
+      "id":"TE700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Indigenous",
+        "entity2type":"tribe",
+        "entity2name":"T'epot'aha'l",
+        "labelRotate":45,
+        "labelScale":2.5,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Salinan",
+        "startdatestr":"700",
+        "enddatestr":"1768"},
+      "geometry":{
+        "type":"Polygon",
+        "coordinates":[[[-121.5, 36.5],[-121.5, 35],[-120.5, 35],[-120.5, 36.5]]]}},
+    { "type":"Feature",
+      "id":"CH700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Indigenous",
+        "entity2type":"tribe",
+        "entity2name":"Chumash",
+        "labelY":5,
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Chumash_people",
+        "startdatestr":"700",
+        "enddatestr":"1768"},
+      "geometry":{
+        "type":"Polygon",
+        "coordinates":[[[-120.75, 35],[-120.75, 34],[-118, 34],[-118, 35]]]}},
+    { "type":"Feature",
+      "id":"YO700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Indigenous",
+        "entity2type":"tribe",
+        "entity2name":"Yokuts",
+        "labelRotate":45,
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Yokuts",
+        "startdatestr":"700",
+        "enddatestr":"1768"},
+      "geometry":{
+        "type":"Polygon",
+        "coordinates":[[[-120.5, 38],[-120.5, 35],[-118, 35],[-118, 38]]]}},
+    { "type":"Feature",
+      "id":"TA700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Indigenous",
+        "entity2type":"tribe",
+        "entity2name":"Tataviam",
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:10:04",
+        "source":"https://en.wikipedia.org/wiki/Tataviam",
+        "startdatestr":"700",
+        "enddatestr":"1768"},
+      "geometry":{
+        "type":"Polygon",
+        "coordinates":[[[-118.5, 34],[-118.5, 33.5],[-117.5, 33.5],[-117.5, 34]]]}},
+    // { "type":"Feature",
```

Comment:
Last change request: please remove the commented section. "True" JSON doesn't allow comments, so this kills some of the scripts under `utilities/` that read the JSON

Reply:
Done. Will push force...